data/nav.yml

```diff
@@ -224,8 +224,8 @@
           path: "pipelines/artifacts"
         - name: "Pipeline tags"
           path: "pipelines/tags"
-        - name: "Build retention"
-          path: "pipelines/build-retention"
```

Comment:
Can you add a redirect for this URL to the new one? I believe they're defined in [routes.rb](https://github.com/buildkite/docs/blob/main/config/routes.rb) (Is that right, @dannymidnight?)

Reply:
Yeah that looks right, I've added in the redirect